src/operations/blas3/gemm_interleaved.hpp

```diff
@@ -157,8 +158,14 @@ class Gemm<input_t, output_t, /* DoubleBuffer = */ false, /* NbcA = */ false,
   const index_t ldb_;
   const index_t ldc_;
   const index_t batch_size_;
+  index_t stridea_;
+  index_t strideb_;
+  index_t stridec_;
+  // Matrices Strides are of no use interleaved
```

Comment:
Did you plan on removing this comment or did you put it here intentionally?

Reply:
Comment removed. Thanks.